base/util.jl

```diff
@@ -276,21 +276,21 @@ See also [`@time`](@ref), [`@timev`](@ref), [`@elapsed`](@ref), and
 [`@allocated`](@ref).
 
 ```julia-repl
-julia> val, t, bytes, gctime, memallocs = @timed rand(10^6);
+julia> o = @timed rand(10^6);
 
-julia> t
+julia> o.time
 0.006634834
 
-julia> bytes
+julia> o.bytes
 8000256
 
-julia> gctime
+julia> o.gctime
 0.0055765
 
-julia> fieldnames(typeof(memallocs))
+julia> fieldnames(typeof(o.gcdiff))
```

Comment:
How about `gcstats` as the name of this field?

Reply:
Right, the rainfall about using a named tuple is that now the names are significant.